libs/datasets/timeseries.py

```diff
@@ -770,6 +786,11 @@ def from_timeseries_df(timeseries_df: pd.DataFrame) -> "MultiRegionDataset":
         return MultiRegionDataset(timeseries=timeseries_df)
 
     def add_fips_static_df(self, latest_df: pd.DataFrame) -> "MultiRegionDataset":
+        # This function is only called on empty datasets and is very unlikely to get any new
+        # callers. This is a useful constraint when simplifying add_static_values.
+        # TODO(tom): Uncomment after fixing test_multi_region_to_from_timeseries_and_latest_values
+        # assert self.timeseries_bucketed.empty
```

Comment:
What's the plan for this TODO?  Upcoming PR?  Maybe sometime?  Other?

Reply:
I don't really have a plan. Upcoming maybe sometime when I don't feel like I'm scrambling to do something more urgent.